### ISSUE TITLE
undefined `sequenceConfig` fails

### ISSUE THREAD

**Issue by timelyportfolio** [knsv/mermaid#109]
It appears there is a bug in this [line](https://github.com/knsv/mermaid/blob/48500652b3b88b1655aa5d7d55e2aabde08f7e16/src/main.js#L25).  I believe it should be 

```
    if (typeof sequenceConfig !== 'undefined') {
      seq.setConf(JSON.parse(sequenceConfig));
    }
```

rather than the existing

```
    if (sequenceConfig !== 'undefined') {
      seq.setConf(JSON.parse(sequenceConfig));
    }
```

**Comment by knsv** [comment]
Thanks! Must have come along with the sequence config feature. Should be fixed with the lastest commit. (Not released yet though).

**Comment by knsv** [comment]
A fix has been included in release 0.3.3
